src/sentry/consumers/__init__.py

```diff
@@ -206,6 +212,9 @@ def ingest_replay_recordings_options() -> List[click.Option]:
         "static_args": {
             "ingest_profile": "release-health",
         },
+        "dlq_topic": settings.KAFKA_INGEST_METRICS_DLQ,
```

Comment:
I don't think it's safe to include this in the same initial PR (it's the part that starts producing)... unless you've ensured the topic exists everywhere?

Reply:
I kept it because `enable_dlq` is set to False, but I will remove it

Reply:
👍